bgpd/bgp_main.c

```diff
@@ -297,6 +301,7 @@ static int bgp_vrf_disable(struct vrf *vrf)
 
 static void bgp_vrf_init(void)
 {
+	ns_init();
```

Comment:
The abstraction we've choosen is that the ns initialization owns the vrf initialization in other places in the code.  Why is this different?